prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{display:flex; min-height: 100vh; align-items:center; justify-content:center;}
```

Comment:
Hey Bryan, I would recommend formatting you CSS like this
```css
    #card-deck {
      min-height: 100vh;
      display: flex;
      align-items: center;
      justify-content: center;
    }
```
as it makes it much easier to read for other developers and allows for lots of properties without having to scroll to the right in the code.

good work on the functionality!

Reply:
copy that thanks